securedrop_client/storage.py

```diff
@@ -135,36 +135,6 @@ def update_local_storage(session: Session,
         update_replies(remote_replies, get_local_replies(session), session, data_dir)
 
 
-def update_source_key(gpg: GpgHelper, local_source: Source, remote_source: SDKSource) -> None:
-    """
-    Updates a source's GPG key.
-    """
-    if not remote_source.key.get("fingerprint"):
-        logger.error("New source data lacks key fingerprint")
-        return
-
-    if not remote_source.key.get("public"):
-        logger.error("New source data lacks public key")
-        return
-
-    if (
-        local_source.fingerprint == remote_source.key['fingerprint'] and
-        local_source.public_key == remote_source.key['public']
-    ):
-        logger.debug("Source key data is unchanged")
-        return
-
-    try:
-        # import_key updates the source's key and fingerprint, and commits
-        gpg.import_key(
-            remote_source.uuid,
-            remote_source.key['public'],
-            remote_source.key['fingerprint']
-        )
-    except CryptoError:
-        logger.error('Failed to update key information for source %s', remote_source.uuid)
-
-
 def update_sources(gpg: GpgHelper, remote_sources: List[SDKSource],
```

Comment:
if we're no longer using GpgHelper in this function, we should remove it

Reply:
:+1: Pulling that thread was fun, but it's gone now.